### PR TITLE
Exhibition guides pages

### PR DESCRIPTION
**Pull request by gestchild**
References #8136 

Sets up the routing for the ~digital~exhibition-guides and adds some skeleton page templates.

~URLs are up for discussion, current thinking is:~

- ~/digital-guides/ (Landing page for all digital guides)~
- ~/digital-guides/{id} (Landing page for particular digital guide)~
- ~/digital-guides/{id}/audio-with-description~
- ~/digital-guides/{id}/audio-without-description~
 - ~/digital-guides/{id}/bsl~
- ~/digital-guides/{id}/captions-and-transcripts~
 
Following[ our discussion](https://wellcome.slack.com/archives/CUA669WHH/p1658323877677509) the URL structure is:
 - /guides/exhibitions
 - /guides/exhibitions/{id}
 - /guides/exhibitions/{id}/{content-type}
 - /guides/exhibitions/{id}/{content-type}#{stop}

We'll need to look at the current guide landing page and rerouting/redirecting the current guide that exists, but that can be dealt with separately.

N.B. not sure why we have 2 link-resolver files that look to be doing the same thing. Will look at that separately.

### CODE REVIEW

**Comment by gestchild** [comment]
closing in favour of #8171

File: content/webapp/app.ts [hunk 1]
```diff
@@ -118,6 +118,13 @@ const appPromise = nextApp
     route('/newsletter', '/newsletter', router, nextApp);
 
     route('/guides', '/guides', router, nextApp);
+    route('/guides/exhibitions', '/exhibition-guides', router, nextApp);
+    route(
+      `/guides/exhibitions/:id(${prismicId})/:type?`,
+      '/exhibition-guide',
+      router,
+      nextApp
+    ); // :type(${guideType})
```
Comment (davidpmccormick) [inline content/webapp/app.ts:127]: Did you decide against having a separate template for the `/:type?` route?

Comment (gestchild) [reply content/webapp/app.ts:127]: it's still up for grabs, this is just a first pass

File: content/webapp/pages/exhibition-guides.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    // const exhibitionGuides = transformQuery(exhibitionGuidesQuery, transformExhibitionGuide);
+    // const jsonLd = exhibitionGuides.results.map(exhibitionGuidesLd);
+    // const basicExhibitionGuides = {
+    //   ...exhibitionGuides,
+    //   results: exhibitionGuides.results.map(transformExhibitionGuideToExhibitionGuideBasic),
+    // };
+
+    const serverData = await getServerData(context);
+
+    // return {
```
Comment (melanierogan) [inline content/webapp/pages/exhibition-guides.tsx:46]: Just wanted to check that the commented out code is related to the TODOs e.g. querying Prismic for the exhibition-guides content that follow in future PRs.

**Review by melanierogan — APPROVED** [review]
The routing approach looks solid, just had a minor comment on the commented out code.

File: content/webapp/pages/exhibition-guides.tsx (hunk 2 above)
Comment (gestchild) [reply content/webapp/pages/exhibition-guides.tsx:49]: yep

**Pull request merged**